### ISSUE TITLE
Some parts of FAQ are translated into Traditional Chinese on Production but not Staging

### ISSUE THREAD

**Issue by jwzimmer** [LLK/scratch-www#1334]
**TL;DR:** Two paragraphs of the FAQ have Traditional Chinese translations available on Production, but not on Staging. I don't know of any reason to expect this. Possibly caused by #1305 (if the English string is being used despite an available localized string).

If this is a bug/ there's no reason this actually is the intended behaviour, then this is probably a blocker, as we really don't want to prevent localized strings from being displayed when they do exist.

To repro:
- View FAQ page on Staging and Production
- Set Language to Traditional Chinese
- Scroll through the questions to see which have been translated

From https://github.com/LLK/scratch-www/pull/1305#issuecomment-304305662.

**FAQ questions in English on Staging but not Production (for Traditional Chinese)**
<img width="718" alt="screen shot 2017-05-31 at 4 49 23 pm" src="https://cloud.githubusercontent.com/assets/1320456/26653560/6f998080-4621-11e7-9290-0f3881409a3e.png">
<img width="655" alt="screen shot 2017-05-31 at 4 49 16 pm" src="https://cloud.githubusercontent.com/assets/1320456/26653557/6f904a2e-4621-11e7-9eb9-36f50a66f613.png">
<img width="899" alt="screen shot 2017-05-31 at 4 48 31 pm" src="https://cloud.githubusercontent.com/assets/1320456/26653558/6f924a0e-4621-11e7-8cd9-9524ebf97cdf.png">
<img width="929" alt="screen shot 2017-05-31 at 4 48 20 pm" src="https://cloud.githubusercontent.com/assets/1320456/26653924/c2894cd4-4622-11e7-8c61-e4f0051c2711.png">

**Comment by chrisgarrity** [comment]
It's not a blocker - we changed the text on the FAQ for those two sections in this build, so translators need to update the translations. I checked in Transifex and the last FAQ question was updated 3 days ago, but it was too recent to get included when we built the release. So those strings didn't have translations and correctly have English as the default.

**Comment by jwzimmer** [comment]
Awesome, thanks @chrisgarrity! 👍
